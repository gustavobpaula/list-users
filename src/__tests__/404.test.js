import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Page404 from '../pages/404';

describe('Testing 404 Page Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Page404 />,
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
