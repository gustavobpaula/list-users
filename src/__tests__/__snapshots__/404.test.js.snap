// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing 404 Page Component should render correctly 1`] = `<NotFound />`;
